crop-me/src/main/java/com/mikelau/croperino/Croperino.java

```diff
@@ -20,14 +20,45 @@
  */
 public class Croperino {
 
-    public static void runCropImage(File file, Activity ctx, boolean isScalable, int aspectX, int aspectY, int color, int bgColor) {
+    public enum ScreenOrientation {
```

Comment:
Crashes for other orientations other than portrait, Merged but will not integrate as of the moment. Thanks for the initiative.